doc/luigi_patterns.rst

```diff
@@ -50,10 +50,10 @@ crashing or lacking their required dependencies for more than a day
 though, which would lead to a missing deliverable for some date. Oops.
 
 To ensure that the above AllReports task is eventually completed for
-every day (value of date parameter), one could e.g. add a loop in
-requires method to yield dependencies on the past few days preceding
-self.date. Then, so long as Luigi keeps being invoked, the backlog of
-jobs would catch up nicely after fixing intermittent problems.
+every day (given by the value of date parameter), one could, e.g., add a loop in
+the `requires` method to yield dependencies on the past few days preceding
```

Comment:
This is an `.rst` file and now `.markdown`. Are you sure you don't want double back-ticks?